readthedocs/builds/views.py

```diff
@@ -84,6 +84,21 @@ def post(self, request, project_slug):
 
 class BuildList(BuildBase, BuildTriggerMixin, ListView):
 
+    def get_queryset(self):
+        # this is used to include only internal version
+        # builds in the build list page
+        self.project_slug = self.kwargs.get('project_slug', None)
+        self.project = get_object_or_404(
+            Project.objects.protected(self.request.user),
```

Comment:
Why you use `.protected` here?

Reply:
@humitos Because it was like this before in the base class. Should I use `. public()` instead